### PR TITLE
Toit language (www.toit.io) highlighting

### PR DESCRIPTION
**Pull request by snxx-lppxx**
### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by joshgoebel** [comment]
Awesome.  Before we could merge this you:

- need to add a proper readme (example: https://github.com/highlightjs/highlightjs-robots-txt) showing how to use your grammar
- highly recommended:  build a CDN distributable (typically in the `dist` folder - see our 3rd party grammar instructions on how to use highlight.js to do this for you)

This way it's easy to use on both the client (as a CDN asset) and server-side (via import).

**Comment by snxx-lppxx** [comment]
Hello @joshgoebel, I've finished. But I don't know what to do next, can I clone the repository into the extra directory as shown in [the documentation][1]? Please, review this!

[1]: https://github.com/highlightjs/highlight.js/blob/main/extra/3RD_PARTY_QUICK_START.md#packaging

**Comment by joshgoebel** [comment]
> can I clone the repository into the extra directory as shown in [the documentation](https://github.com/highlightjs/highlight.js/blob/main/extra/3RD_PARTY_QUICK_START.md#packaging)?

That's how it's typically done to allow us to security check your regex and build your `dist`... once it's there just building Highlight.js `cdn` build should create the `dist` folder automagically.

**Comment by snxx-lppxx** [comment]
Yeah, I have already created this file in my repository:

image

**Comment by snxx-lppxx** [comment]
> Did it not generate the `toit.es.min.js` file as well?

It was, but it was absent in the example, and that's why I removed it.

**Comment by joshgoebel** [comment]
I'd add it.  Someone who is looking for ES6 likely knows how to use it.  I really should update the example though with an ES6 example.  But that shouldn't hold you up.

File: CHANGES.md [hunk 1]
```diff
@@ -5,10 +5,12 @@ Grammars:
 - enh(swift) add SE-0335 existential `any` keyword (#3515) [Bradley Mackey][]
 - enh(swift) add support for `distributed` keyword [Marcus Ortiz][]
 - enh(xml) recognize Unicode letters instead of only ASCII letters in XML element and attribute names (#3256)[Martin Honnen][]
+- Added 3rd party Toit grammar to SUPPORTED_LANGUAGES [snxx][]
 
 [Bradley Mackey]: https://github.com/bradleymackey
 [Marcus Ortiz]: https://github.com/mportiz08
 [Martin Honnen]: https://github.com/martin-honnen
+[snxx]: https://github.com/snxx-lppxx
```
Comment (joshgoebel) [inline CHANGES.md:13]: I assume you don't want your name in the credits?

**Comment by snxx-lppxx** [comment]
By the way, I already added that missing file.

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**